### DISCUSSION TITLE
When is a 2nd detector (eg Coral ) needed? Will 1 coral be sufficient for 20-22 IP cameras?

### DISCUSSION THREAD

**Discussion by epicurean70** [blakeblackshear/frigate#7454] [Ask A Question]
Just wondering if there are indicators in Frigate GUI to let us know that the single coral is sufficient or overwhelmed and need a 2nd one?

I have 20 Dahua IP cameras now

**Comment by NickM-27** [comment]
You would check the `skipped fps` and if that gets considerably high (some skipped fps during high activity is fine) you'd want to consider another. 

However, we discovered in 0.13 that the skipped fps is broken in 0.12

**Reply by epicurean70** [reply]
Is there otherways to figure this out without waiting for 0.13? I understand 0.13 will not be released so soon

**Reply by NickM-27** [reply]
You could look at the detection fps sensor for the `Frigate` device and see how often it is maxed out for the given inference speed

**Reply by epicurean70** [reply]
What would be considered "often" to need a 2nd coral?

**Reply by NickM-27** [reply]
if it only happens at times of peak activity (lots of cars driving around; lots of wind on multiple cameras) then it is not a problem. If you are seeing many times of usual activity through the day then I'd suggest it would likely be worth looking into another.
